lib/mongoose_push/application.ex

```diff
@@ -86,6 +88,39 @@ defmodule MongoosePush.Application do
     end)
   end
 
+  defp metrics do
+    [
+      # Summary is not yet supported in TelemetryMetricsPrometheus
+      Telemetry.Metrics.counter("mongoose_push.push.time",
+        tags: [:status, :service, :type, :reason],
+        tag_values: fn metadata ->
+          case metadata.status do
+            :success ->
+              Map.merge(
+                %{
+                  type: :success,
+                  reason: :success
```

Comment:
`error_reason`?

Reply:
Also, I feel that empty value would be a better match here.

Reply:
Likewise here, I don't get what is it about.